cpp/src/cuda/cuda_matrix_vector_multiply_shared.cu

```diff
@@ -0,0 +1,123 @@
+// The MIT License (MIT)
```

Comment:
There is this file and cuda_matrix_vector_multiply_shared_memory.cu, should we remove this one?

Reply:
I think for now I want to keep it as a baseline for a non-shared memory implementation. When the logistic_regression code works I can remove this.